hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/CallerContext.java

```diff
@@ -140,6 +141,14 @@ public Builder setSignature(byte[] signature) {
       return this;
     }
 
+    public String getContext() {
```

Comment:
I think it would be a good place to have a javadoc describing what the context would look like and give examples.